src/pipeline_migration/actions/modify/generic.py

```diff
@@ -175,17 +185,17 @@ def register_cli(subparser) -> None:
     subparser_replace.add_argument(
         "yaml_path",
         help=(
-            "YAML path (in YAML format). Must point to a sequence or a map item. "
+            "YAML path (in YAML format). Can point to a scalar value. "
```

Comment:
```suggestion
            "YAML path (in YAML format). Must point to a scalar, a sequence or a map item. "
```

Scalar type is different from the sequence and mapping. "sequence or a map item" should not be  replaced with "scalar value".

Reply:
I removed it as Adam suggested